test/python/test_reactor.py

```diff
@@ -1017,11 +1017,11 @@ def test_draw_wall(self):
         self.make_reactors(T1=T1, P1=P1, X1=X1, T2=T2, P2=P2, X2=X2)
         self.r1.name = "Name 1"
         self.r2.name = "Name 2"
-        w = ct.Wall(self.r1, self.r2, U=0.1, edge_attr={'style': 'dotted'})
+        w = ct.Wall(self.r1, self.r2, U=0.1, edge_attr={'style': 'dotted'}, name="wall")
         w.edge_attr = {'color': 'green'}
         graph = w.draw(node_attr={'style': 'filled'},
                        edge_attr={'style': 'dashed', 'color': 'blue'})
-        expected = [('\t"Name 2" -> "Name 1" [label="q = 30 W" '
+        expected = [('\t"Name 2" -> "Name 1" [label="wall\\nq̇ = 30 W" '
```

Comment:
These tests are failing due to the change in case between $\dot q$ and $\dot Q$. I like the capitalized version, but either is fine.

Reply:
Not sure why I didn't pick up on the test failures. I agree with your preference btw, where I had already implemented `Q̇`.